xds/src/main/java/io/grpc/xds/ClusterManagerLoadBalancer.java

```diff
@@ -81,13 +84,12 @@ protected Map<Object, ChildLbState> createChildLbMap(ResolvedAddresses resolvedA
         resolvedAddresses.getLoadBalancingPolicyConfig();
     Map<Object, ChildLbState> newChildPolicies = new HashMap<>();
     if (config != null) {
-      for (Entry<String, PolicySelection> entry : config.childPolicies.entrySet()) {
-        ChildLbState child = getChildLbState(entry.getKey());
+      for (String key : config.childPolicies.keySet()) {
+        ChildLbState child = getChildLbState(key);
         if (child == null) {
-          child = new ClusterManagerLbState(entry.getKey(),
-              entry.getValue().getProvider(), entry.getValue().getConfig());
+          child = new ClusterManagerLbState(key, GracefulSwitchLoadBalancerFactory.INSTANCE, null);
```

Comment:
The only thing this argument is used for is to pass as an argument to `getChildAddresses()`, which by default copies it to the child's ResolvedAddresses. In this change, this specific policy's `getChildAddresses()` ignores the argument.

As I say in the description, I'll be cleaning that up in a follow-up (I think the very next PR):
> In a following commit I'll remove child policy config from the child's state, because it should be updated every address update. At that point unusedChildConfig will go away.

This is a pre-migration to make removing the child policy config from the child's state trivial.